example/lib/examples/responsive_layouts.dart

```diff
@@ -21,13 +23,19 @@ class _ExampleResponsiveLayoutsState extends State<ExampleResponsiveLayouts> {
   void dispose() {
     // This widget state owns the file loader, dispose it.
     fileLoader.dispose();
+    viewModelInstance.dispose();
     super.dispose();
   }
 
+  void _onLoaded(RiveLoaded state) {
+    viewModelInstance = state.controller.dataBind(DataBind.auto());
```

Comment:
Ah didn't notice this last time, you can remove this, and instead provide the `dataBind` property to `RiveWidgetBuilder`.

Then in onLoad, you can do `state.viewModelInstance` to access it